Nodes/ClassNode.cs

```diff
@@ -45,29 +45,32 @@ public IntPtr Address
 
 		public event NodeEventHandler NodesChanged;
 
-		internal ClassNode(bool notifyClassCreated)
+		internal ClassNode(bool notifyClassCreated, IntPtr Address = default(IntPtr))
```

Comment:
Lowercase name.
Add a second constructor with just the bool instead of the optional parameter and call this constructor with IntPtr.Zero then.